src/custom-element.js

```diff
@@ -1362,8 +1362,9 @@ function createBaseCustomElementClass(win) {
      * @package @final @this {!Element}
      */
     getRealChildren() {
-      return dom.childElements(this, element =>
-        !isInternalOrServiceNode(element));
+      return dom.childElementByTag(this, '*').filter(element => {
```

Comment:
This is relatively unclear that `ByTag` can do wildcards... E.g. does `ELement.getElementsByTagName` do wildcards?

Another question: is querying here really faster than first-child -> siblings walk?